src/mongo/MongoTripod.class.php

```diff
@@ -595,12 +598,12 @@ protected function getMongoTripodConfigInstance()
 
 
     /**
-     * Returns the correct observer as per the supported operation
-     * @param $operation string must be either views, tables or search
-     * @return SplObserver
+     * Returns the composite that can perform the supported operation
+     * @param $operation string must be either OP_VIEWS, OP_TABLES or OP_SEARCH
+     * @return IComposite
      * @throws TripodException when an unsupported operation is requested
      */
-    public function getObserver($operation)
+    public function getComposite($operation)
```

Comment:
This method name leaves a little to be desired, but it's still a :+1: just for being better than 'getObserver'